test/unit-tests/helpers.ts

```diff
@@ -134,19 +140,125 @@ describe("helpers", () => {
 			}
 		];
 
-		let assertTestData = (testData: ITestData) => {
-			// getPropertyName accepts function as argument.
-			// The tests will use strings in order to skip transpilation of lambdas to functions.
-			let actualResult = helpers.getPropertyName(testData.input);
-			assert.deepEqual(actualResult, testData.expectedResult, `For input ${testData.input}, the expected result is: ${testData.expectedResult}, but actual result is: ${actualResult}.`);
-		};
-
+		// getPropertyName accepts function as argument.
+		// The tests will use strings in order to skip transpilation of lambdas to functions.
 		it("returns correct property name for ES5 functions", () => {
-			_.each(ES5Functions, assertTestData);
+			_.each(ES5Functions, testData => assertTestData(testData, helpers.getPropertyName));
 		});
 
 		it("returns correct property name for ES6 functions", () => {
-			_.each(ES6Functions, assertTestData);
+			_.each(ES6Functions, testData => assertTestData(testData, helpers.getPropertyName));
+		});
+	});
+
+	describe("toBoolean", () => {
+		let toBooleanTestData: ITestData[] = [
+			{
+				input: true,
+				expectedResult: true
+			},
+			{
+				input: false,
+				expectedResult: false
+			},
+			{
+				input: "true",
+				expectedResult: true
+			},
+			{
+				input: "false",
+				expectedResult: false
+			},
+			{
+				input: "",
+				expectedResult: false
+			},
+			{
+				input: null,
+				expectedResult: false
+			},
+			{
+				input: undefined,
+				expectedResult: false
+			},
+			{
+				input: "",
+				expectedResult: false
+			},
+			{
+				input: "some random text",
+				expectedResult: false
+			},
+			{
+				input: { "true": true },
+				expectedResult: false
+			},
+			{
+				input: {},
+				expectedResult: false
+			},
+			{
+				input: { "a": { "b": 1 } },
+				expectedResult: false
+			}
+		];
+
+		it("returns expected result", () => {
+			_.each(toBooleanTestData, testData => assertTestData(testData, helpers.toBoolean));
+		});
+	});
+
+	describe("isNullOrWhitespace", () => {
+		let isNullOrWhitespaceTestData: ITestData[] = [
+			{
+				input: "",
+				expectedResult: true
+			},
+			{
+				input: "     ",
+				expectedResult: true
+			},
+			{
+				input: null,
+				expectedResult: true
+			},
+			{
+				input: undefined,
+				expectedResult: true
+			},
+			{
+				input: [],
+				expectedResult: false
+			},
+			{
+				input: ["test1", "test2"],
+				expectedResult: false
+			},
+			{
+				input: {},
+				expectedResult: false
+			},
+			{
+				input: { a: 1, b: 2 },
+				expectedResult: false
+			},
+			{
+				input: true,
+				expectedResult: false
+			},
+			{
+				input: false,
+				expectedResult: false
+			}
```

Comment:
You could add a couple more cases like 
```JavaScript
{
    input: '\n',
    expectedResult: false
},
{
    input: '\r\n',
    expectedResult: false
},
{
    input: '\t',
    expectedResult: false
}
```

In order to ensure that the regex won't be changed for example from `/\s/` to simply `/ /`